### PR TITLE
CVSB-3069 Updated test-type taxonomy for psv.

### PR DESCRIPTION
**Pull request by bsiddappa**

### CODE REVIEW

File: tests/resources/test-types.json [hunk 1]
```diff
@@ -1045,7 +1117,7 @@
     "nextTestTypesOrCategories": [
       {
         "id": "38",
-        "linkedIds": ["1","2","3","4","5","6","7","8","10","14","15","16","23","18","19","21","22","27","28","39"],
+        "linkedIds": ["1","2","3","4","5","6","7","8","10","14","15","16","23","18","19","21","22","27","28","39","41"],
```
Comment (vladddragomir) [inline tests/resources/test-types.json]: you should also add the category's id which is 40

File: tests/resources/test-types.json [hunk 2]
```diff
@@ -1081,7 +1153,7 @@
       },
       {
         "id": "39",
-        "linkedIds": ["1","2","3","4","5","6","7","8","10","14","15","16","23","18","19","21","22","27","28","38"],
+        "linkedIds": ["1","2","3","4","5","6","7","8","10","14","15","16","23","18","19","21","22","27","28","38","41"],
```
Comment (vladddragomir) [inline tests/resources/test-types.json]: same here

File: tests/resources/test-types.json [hunk 3]
```diff
@@ -778,6 +788,68 @@
...
+                "forVehicleAxles": null,
+                "defaultTestCode": "wil",
+                "linkedTestCode": null
+              }
+            ]
+          }
+        ]
+      },
+      {
+        "id": "40",
```
Comment (vladddragomir) [inline tests/resources/test-types.json]: 40 and 41 are already used by hgvs and trailers test-types. you should replace them with 92 and 93

File: tests/resources/test-types.json (hunk 1 above)
Comment (bsiddappa) [reply tests/resources/test-types.json]: I dont see 11,12,13 added but categories -14, 15,16 added. Same is the case with 20, 26 which are not added, but (21,22) and (27,28) added here. Kindly verify and let me know if both 92 and 93 is required to be added here.

File: tests/resources/test-types.json (hunk 3 above)
Comment (bsiddappa) [reply tests/resources/test-types.json]: Agree, will make changes.

File: tests/resources/test-types.json (hunk 1 above)
Comment (bsiddappa) [reply tests/resources/test-types.json]: Done.

**Pull request merged**